test/specs/utils/paginator.js

```diff
@@ -0,0 +1,50 @@
+/* eslint no-multi-spaces: "off" */
+/* eslint space-in-parens: "off" */
+/* eslint brace-style: "off" */
+/* eslint max-statements-per-line: "off" */
+var _ = require('lodash');
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var Paginator = require('../../../lib/utils/paginator');
+
+describe('Paginator', function () {
+  describe('Auto height (Terminal mode)', function () {
+    var paginator;
+    var height;
+
+    beforeEach(function () {
+      process.stdout.getWindowSize = sinon.stub().returns([80, 20]);
```

Comment:
This doesn't get restored (same with other test).

In this case you'd want `sinon.stub(process.stdout, 'getWindowSize').returns([80, 20])`